src/common/routes.ts

```diff
@@ -0,0 +1,3 @@
+export const narrativeSelectedPath = '/narratives/:id/:obj/:ver';
```

Comment:
It's used in `features/navigator/common.ts` as well as `app/Routes.tsx` so I put it in `common/routes.ts`. I put it here originally to avoid any circular imports, but we can move it if desired.

Reply:
Import into `Navigator.tsx` and then export from there and for `app/Routes.tsx` import from `Navigator.tsx`.